src/xpk/commands/cluster.py

```diff
@@ -87,6 +89,22 @@ def cluster_create(args) -> None:
   if create_cluster_command_code != 0:
     xpk_exit(create_cluster_command_code)
 
+  # Enable WorkloadIdentity if not enabled already.
+  if args.enable_workload_identity or args.enable_gcsfuse_driver:
```

Comment:
Nit: curious if you want to check for the case `if not args.enable_workload_identity and args.enable_gcsfuse_driver: ... return error`

Reply:
does false value for args.enable_workload_identity means user explicitly set it to false? or it can also mean user did not provided the flag?
I assumed the latter is the case, and we'll automatically enable WI if gsfuse is enabled.
What do you think?